modules/general/stackoverflow.py

```diff
@@ -0,0 +1,29 @@
+import gzip
+import json
+import urllib.request
+
+import glados
+
+
+STACK_EXCHANGE_API = 'https://api.stackexchange.com/2.2/search/advanced?'
+
+class StackOverflow(glados.Module):
+    def get_help_list(self):
+        return [
+            glados.Help('so', '<search terms>', 'Searches stuff in stackoverflow')
+        ]
+
+    @glados.Module.commands('so')
+    def search(self, message, content):
+        query_string = 'order=desc&sort=activity&site=stackoverflow&pagesize=1&q={}'.format(content.replace(' ', '+'))
```

Comment:
You should be using python's urlencode() for generating URLs. As it is now, it presents a potential security risk. Something along the lines of:
```python
from urllib.parse import urlencode
f = {
    'order': 'desc',
    'sort': 'activity',
    'site': 'stackoverflow',
    'pagesize': '1',
    'q': content
}
query_string = urlencode(f)
```